apply/run.go

```diff
@@ -47,23 +47,36 @@ func IsIPList(args string) bool {
 	return true
 }
 
-func IsCidrString(arg string) bool {
+func IsCidrString(arg string) error {
```

Comment:
The function definition should be improved. If you use `IsCidrString`, then it should return a bool. If you define it as `ParseCIDR`, other use `ParseCIDR` directly, why to define another function?

Reply:
job done.